tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/MediaType/MediaType.spec.ts

```diff
@@ -0,0 +1,538 @@
+import {expect} from "@playwright/test";
+import {AliasHelper, ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+
+// TODO: Remove this after the tests have passed on the pipeline
+test.describe('Media Type tests @smoke', () => {
```

Comment:
We will remove the test.describe after the test has passed on the pipeline. So it would be unnecessary to split the Design and Structure tab into different files.

Reply:
Similar to the Document Type tests, there are a lot of tests in "MediaType" test class. Separating it into 3 files would make the code clearer and easier to maintain.